Phoenix/hello/lib/hello_web/templates/hello/index.html.heex

```diff
@@ -0,0 +1,3 @@
+<section class="phx-hero">
+    <h2>Hello World, from Phoenix!</h2>
+</section>
```

Comment:
@samu975 missing blank line at the end of the file. TIP: You can configure your text editor to handle this automatically for you!